app/models/container_group.rb

```diff
@@ -37,6 +40,18 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def waiting_containers
+    "#{containers.to_a.count { |container| container.state.eql? 'waiting' }}/#{containers.count}"
```

Comment:
This looks overly complex wrapped in a string.  Better to find the count and covert to a string.
Why are these counts stringy anyway, instead of numeric?